source/faq.rst

```diff
@@ -0,0 +1,98 @@
+.. _faq:
+
+================================================================================
+Frequently Asked Questions
+================================================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Why do we need to follow "the Kubernetes way" when Kubernetes was never intended to run databases?
+=====================================================================================================
+
+As it is well known, the Kubernetes approach is targeted at stateless
+applications but provides ways to store state (in Persistent Volumes, etc.) if
+the application needs it. Generally, a stateless mode of operation is supposed
+to provide better safety, sustainability, and scalability, it makes the
+already-deployed components interchangeable. You can find more about substantial
+benefits brought by Kubernetes to databases in `this blog post <https://www.percona.com/blog/2020/10/08/the-criticality-of-a-kubernetes-operator-for-databases/>`_.
+
+The architecture of state-centric applications (like databases) should be
+composed in a right way to allow avoiding crashes, data loss, or data
+inconsistencies during hardware fail. It is the main goal of the database
+Kubernetes Operator to solve or simplify these aspects.
+
+It worth to mention, that in a non-Kubernetes environment issues are the same.
+Unfortunately, we, Engineers, tend to think that crash, data loss, or data
+inconsistency issues are not important or even do not exist in the
+non-Kubernetes world, but of course that's not true.
+
+Percona Kubernetes Operator for Percona XtraDB Cluster uses synchronous
+replication which guarantees consistency on any kind of fail of one cluster
+member. It has automatic recovery if one member had fully or partially lost
+data. The proxy layer automatically switches traffic to alive members. The
+Operator provides this among other things which are typically not solved at all
+in a typical non-Kubernetes setup.
```

Comment:
And lets remove this completely. I believe it is covered by previous abstract. WDYT?